nautobot/docs/user-guide/platform-functionality/secret.md

```diff
@@ -30,8 +28,6 @@ Each Secret is associated with a secrets provider (not to be confused with a cir
 - *Environment Variable* - for retrieving a secret value defined in an environment variable; Secrets using this provider must specify the `variable` name to retrieve.
 - *Text File* - for retrieving a secret value stored in a text file; Secrets using this provider must specify the absolute `path` of the file to retrieve.
 
-+/- 1.4.3
-    When using the Text File secrets provider, any leading and trailing whitespace or newlines will be stripped.
```

Comment:
Change to a `!!! tip` instead of removing?

Reply:
makes sense, I added a tip here for that statement